engine/src/main/java/org/terasology/rendering/dag/nodes/ApplyDeferredLightingNode.java

```diff
@@ -77,8 +77,6 @@ public void process() {
 
         // Actual Node Processing
 
-        glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT); // TODO: this is necessary - but why? Verify in the shader.
```

Comment:
This was only required because the shader wasn't being cleared even once, and that lead to problems with depth buffer. Moved this task to a more appropriate location (in WorldRendererImpl, in it's own BufferClearingNode)